src/compute/src/server.rs

```diff
@@ -316,6 +322,21 @@ pub async fn compute_node_serve(
     let config_srv = ConfigServiceImpl::new(batch_mgr, stream_mgr);
     let health_srv = HealthServiceImpl::new();
 
+    let telemetry_manager = TelemetryManager::new(
+        system_params_manager.watch_params(),
+        Arc::new(meta_client.clone()),
+        Arc::new(ComputeTelemetryCreator::new()),
+    );
+
+    // if the toml config file or env variable disables telemetry, do not watch system params change
+    // because if any of configs disable telemetry, we should never start it
+    if config.server.telemetry_enabled && telemetry_env_enabled() && telemetry_enabled {
```

Comment:
I think we should stll `watch_params_change` if the system param `telemetry_enabled` is false given that it can be changed in runtime.

Reply:
So, we always start watching system_params If both the config and the environment variable are true.
```rust
if config.server.telemetry_enabled && telemetry_env_enabled(){
   // start watching 
}
```